src/pages/BlogPage/index.tsx

```diff
@@ -0,0 +1,266 @@
+
+  const socialLinks = [
+    {
+        label: "Facebook",
+        imgURL: "../Images/logos/facebook_blue.svg",
+    },
+    {
+        label: "Slack",
+        imgURL: "../Images/logos/slack_blue.svg",
+    },
+    {
+        label: "LinkedIn",
+        imgURL: "../Images/logos/linkedin_blue.svg",
+    },
+    {
+        label: "Twitter",
+        imgURL: "../Images/logos/twitter_blue.svg",
+    },
+  ];
+
+  const handleSocialSharing = (label:string) => {
+    switch(label) {
+      case "Facebook":
+        window.open(
+          `http://www.facebook.com/sharer.php?u=${currentLocation}`, "_blank");
+        break;
+      case "LinkedIn":
+        window.open(
+          `https://www.linkedin.com/shareArticle?mini=true&amp;url=${currentLocation}&amp;title=${metadata.title}`, "_blank");
+        break;
+      case "Twitter":
+        window.open(
+          `https://twitter.com/intent/tweet?original_referer=${currentLocation}&amp;text=${metadata.title};url=${currentLocation}`, "_blank");
+        break;
+    }
+  };
+
+  return (
+    <>
+      <Grid
+        container
+        direction="row"
+        justify="center"
+        alignItems="center"
+      >
+        {content ? (
+          <Grid item xs={12}>
+            <div className={classes.blogHeader}>
+              {(width > mobileBreakpoint) &&
+                <Breadcrumbs aria-label="breadcrumb" className={classes.breadCrumbs}>
+                  <Link color="inherit" href="/blog">
+                    {t('blog.blog')}
+                  </Link>
+                  <Link color="inherit" href={`/blog/${currentBlogDetails.slug}`}>
+                      {metadata.title}
+                  </Link>
+                </Breadcrumbs>
+              }
+            <ReactMarkdown children={metadata.title} className={classes.blogTitle} />
+              <div>
+              <div className={classes.container}>
+                  
+                    <div className={classes.author}>
+                      <div className={classes.authorImg}>
+                        <img src={`../Images/blog/authors/${metadata.author}.png`} alt={metadata.author}></img>
+                      </div>
+                      <div className={classes.date}>
+                        <ReactMarkdown children={metadata.author} className={classes.authorName} />
+                        <div className={classes.dateAndTimeWrapper}><ReactMarkdown children={metadata.date} />  / {readingTime(content)} {t('blog.minToRead')}</div>
+                      </div>
+                    </div>
+                  
+                  
+                    <div className={classes.shareWrapper}>
+                      <Typography className={classes.share}>{t('blog.share')}</Typography>
+                      <div className={classes.socialIconsWrapper}>
+                          {socialLinks.map(({ label, imgURL }) => {
+                              return (   
+                                (label === "Slack") ? 
```

Comment:
const SLACK = "Slack" could be a better approach as we are using it multiple times.

Reply:
Done